.travis.yml

```diff
@@ -1,11 +1,11 @@
-sudo: false
 language: python
 python:
   - "nightly"
   - "3.7-dev"
   - "3.6.2"
   - "3.6.1"
   - "3.6.0"
+  - "3.5.4"
```

Comment:
I don't think it's good use of resources to test every bugfix version of 3.5. We started testing 3.5.1 specifically because that's the version used on Ubuntu 16.04 (their LTS release, which Dropbox uses for CI). Though now it seems they updated to 3.5.2. If you really want 3.5.4, let's remove 3.5.3.

Reply:
Now I understand.  Travis is running these tests on Ubuntu 14.04 which EOLs this month.  https://wiki.ubuntu.com/Releases  If Ubuntu 16.04 is desirable then add __dist: xenial__ but I doubt that Python 3.3 will work on Xenial.  Thanks for the review.